identity-wallet/src/state/mod.rs

```diff
@@ -138,6 +138,10 @@ impl AppState {
     }
 }
 
+fn hash(url: &str) -> String {
```

Comment:
This should be moved to either `identity-wallet/src/state/core_utils/helpers.rs` or `identity-wallet/src/persistence.rs`. I prefer the first since although this helper fn is mainly concerned with the assets and thos functions live in the persistence file, this helper is not actually used in that file but throught multiple reducers so it should be in core_utils/helpers i think.